packages/api/src/command/medical/document/document-webhook.ts

```diff
@@ -56,7 +56,7 @@ export const processPatientDocumentRequest = async (
     };
     const webhookRequest = await createWebhookRequest({ cxId, type: whType, payload });
     // send it to the customer and update the request status
-    await processRequest(webhookRequest, settings);
+    if (!patient.data.whOverride) await processRequest(webhookRequest, settings);
```

Comment:
We should cancel the flow before we create the WH request on our DB, right?

Here and on the consolidated one

Reply:
No @leite08 the idea is that when we internally use the endpoint, we can access the results in the DB but the customer doesn't get the webhook request results.